### PR TITLE
Adjust PyTorch Dali pipeline to be similar to MXNet example

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
python -m torch.distributed.launch --nproc_per_node=8 main.py -a resnet50 --fp16 --b 128 --static-loss-scale 128.0 --workers 4 --lr=0.4 ./
Prec@1 76.202 Prec@5 93.027
I think it is ready to go.

File: docs/examples/pytorch/pytorch-resnet50.rst [hunk 1]
```diff
@@ -3,16 +3,16 @@ ImageNet training in PyTorch
 
 This implements training of popular model architectures, such as ResNet, AlexNet, and VGG on the ImageNet dataset.
 
-This version has been modified to use DALI. It assumes that the imagenet dataset is in a Caffe2 lmdb format.
+This version has been modified to use DALI. It assumes that the imagenet dataset is in a Caffe lmdb format.
```
Comment (ptrendx) [inline docs/examples/pytorch/pytorch-resnet50.rst]: Raw JPEGs from ImageNet dataset.

Comment (JanuszL) [reply docs/examples/pytorch/pytorch-resnet50.rst]: Done
